api/proto/profile/local_profile.proto

```diff
@@ -54,3 +54,18 @@ message RadioConfig {
    bool radio_silence = 2;
 }
 
+// DeviceInfo message is sent in the POST request to the api/v1/info API.
+message DeviceInfo {
```

Comment:
I think we should scope this to just AppInfo.

Reply:
done